### ISSUE TITLE
Feature to show graph of grades over time

### ISSUE THREAD

**Issue by MohanMaker** [Aspine/aspine#200]
I think it would be cool to add a graph that plots your average GPA from each quarter over time, and calculates a trendline. This will be easier to do when we have the API.

**Comment by jadebuckwalter** [comment]
@MohanMaker I really like this idea! An idea I had for a related feature would be to graph grades in particular subjects over time. For example, there would be a graph that would show your grades in all of the math classes you've taken, a different one for history classes, etc, and give you a trendline for each subject. When you hover over a point on a graph, it would tell you the class name and your grade for that quarter. This could be done by collecting the course IDs in addition to the class names, because each course has an ID with a letter representing the subject area (M for math, S for science, etc.) and a number representing the course.

**Comment by psvenk** [comment]
I removed the "on hold" label, but then added it back because it would make sense to do this after #54 is closed.
